### PR TITLE
update trends 4

### PR DESCRIPTION
**Pull request by katamaks**
@knorrie

### CODE REVIEW

File: content/developerportal/operate/trends-v4.md [hunk 1]
```diff
@@ -55,39 +55,45 @@ The requests graph shows the number of requests that are sent from the client or
 
 The request types are:
```
Comment (jtwaleson) [inline content/developerportal/operate/trends-v4.md]: Commonly used types are:

File: content/developerportal/operate/trends-v4.md [hunk 2]
```diff
@@ -55,39 +55,45 @@ The requests graph shows the number of requests that are sent from the client or
...
-*   update: Amount of SQL UPDATE statements per second. The SQL UPDATE Query is used to modify the existing records in a table.
-*   select: Amount of SQL SELECT statements per second. The SQL SELECT statement is used to fetch the data from a database table which returns this data in the form of a result table.
-*   delete: Amount of SQL DELETE statements per second. The SQL DELETE Query is used to delete the existing records from a table.
+Type | Explanation
+------------ | -------------
+inserts | Amount of SQL INSERT INTO statements per second. This is used to add new rows of data to a table in the database.
+transactions | Amount of SQL transactions per second. A transaction is a unit of work that is performed against a database.
+update | Amount of SQL UPDATE statements per second. The SQL UPDATE Query is used to modify the existing records in a table.
+select | Amount of SQL SELECT statements per second. The SQL SELECT statement is used to fetch the data from a database table which returns this data in the form of a result table.
+delete | Amount of SQL DELETE statements per second. The SQL DELETE Query is used to delete the existing records from a table.
```
Comment (jtwaleson) [inline content/developerportal/operate/trends-v4.md]: DELETE, INSERT INTO, UPDATE, SELECT should have backticks around them: \`\`

File: content/developerportal/operate/trends-v4.md [hunk 3]
```diff
@@ -55,39 +55,45 @@ The requests graph shows the number of requests that are sent from the client or
...
+api-doc/ | A general api doc overview for other doc request handlers
+debugger/ | Mendix runtime debugger request handler
+/ | Default request handler
+FileDocuments | Request handler used for serving files
+odata-docs/ | documentation request handler for odata
+openid/ | OPENID authentication request handler
+p/ | Request handler for Custom page urls
+rest-doc/ | HTTP Rest webservice request handler documentation
+ws-doc/ | SOAP webservice request handler documentation
+ws | SOAP webservice call request handler
```
Comment (jtwaleson) [inline content/developerportal/operate/trends-v4.md]: ws/ (add trailing slash)

File: content/developerportal/operate/trends-v4.md (hunk 1 above)
Comment (katamaks) [inline content/developerportal/operate/trends-v4.md]: done

File: content/developerportal/operate/trends-v4.md (hunk 3 above)
Comment (katamaks) [inline content/developerportal/operate/trends-v4.md]: done

File: content/developerportal/operate/trends-v4.md (hunk 2 above)
Comment (katamaks) [inline content/developerportal/operate/trends-v4.md]: done

**Comment by katamaks** [comment]
@jtwaleson please approve review

**Pull request merged**